### DISCUSSION TITLE
Koinu and Atomic Swap :)

### DISCUSSION THREAD

**Discussion by qlpqlp** [dogecoin/dogecoin#2751] [Help needed]
GM to all Devs :)

Billy Markus and Richard XL has talked to me about this 2 options available.

I have at night, made some research, but there is little to none available (after pit kids to school will search more).

Can someone point me to the right direction of both?

Koinu to Mint/Transfer NFTs (aka, send transactions with data)

Atomic Swap to prove its the owner and/or sell an NFT and/or auction sell using time lock wen the auction is active for the highest bid :)

Thanks in advance.

**Comment by patricklodder** [comment]
It's very hard to understand what these guys mean without knowing what they said if you're not them. What problem are we trying to solve, exactly?

**Reply by qlpqlp** [reply]
Using a simple Koinu to "represent" an asset or a quantity of an asset, just exploring what can be done here nothing else :)

For rhe Atomic Swap, 2 toughts:

1- Use to execute a script between 2 persons/address's to validate first before the payment

2 - For an online store and/or fisical store POS, for executing some actions to both or none acept the payment or refuse the payment

always on the same chain, Dogecoin, not croaa chain :)
